website/docs/guides/core-to-cloud-1.md

```diff
@@ -15,44 +16,50 @@ recently_updated: true
-- [Developer setup:](https://docs.getdbt.com/guides/core-to-cloud-1?step=6#developer-setup) Understand the setup needed for developing in dbt Cloud.
-- [Environment variables](https://docs.getdbt.com/guides/core-to-cloud-1?step=7#environment-variables): Discover how to manage environment variables in dbt Cloud, including their priority.
-- [Orchestration setup](https://docs.getdbt.com/guides/core-to-cloud-1?step=8#orchestration-setup): Learn how to prepare your dbt Cloud environment and jobs for orchestration.
-- [Models configuration](https://docs.getdbt.com/guides/core-to-cloud-1?step=9#models-configuration): Get insights on validating and running your models in dbt Cloud, using either the dbt Cloud IDE or dbt Cloud CLI.
-- [What's next?](https://docs.getdbt.com/guides/core-to-cloud-1?step=10#whats-next): Summarizes key takeaways and introduces what to expect in the following guides.
+- [Account setup](https://docs.getdbt.com/guides/core-to-cloud-1?step=4): Learn how to create a dbt Cloud account, invite team members, and configure it for your team.
+- [Data platform setup](https://docs.getdbt.com/guides/core-to-cloud-1?step=5): Find out about connecting your data platform to dbt Cloud.
+- [Git setup](https://docs.getdbt.com/guides/core-to-cloud-1?step=6): Learn to link your dbt project's Git repository with dbt Cloud.
+- [Developer setup:](https://docs.getdbt.com/guides/core-to-cloud-1?step=7) Understand the setup needed for developing in dbt Cloud.
+- [Environment variables](https://docs.getdbt.com/guides/core-to-cloud-1?step=8): Discover how to manage environment variables in dbt Cloud, including their priority.
+- [Orchestration setup](https://docs.getdbt.com/guides/core-to-cloud-1?step=9): Learn how to prepare your dbt Cloud environment and jobs for orchestration.
+- [Models configuration](https://docs.getdbt.com/guides/core-to-cloud-1?step=10): Get insights on validating and running your models in dbt Cloud, using either the dbt Cloud IDE or dbt Cloud CLI.
+- [What's next?](https://docs.getdbt.com/guides/core-to-cloud-1?step=11): Summarizes key takeaways and introduces what to expect in the following guides.
+
+### Related docs
+- [Learn dbt Cloud](https://courses.getdbt.com/collections) on-demand video learning.
+- Book [expert-led demos](https://www.getdbt.com/resources/dbt-cloud-demos-with-experts) and insights
+- Work with the [dbt Labs’ Professional Services](https://www.getdbt.com/dbt-labs/services) team to support your data organization and migration.
 
 ## Prerequisites
 
 - You have an existing dbt Core project connected to a Git repository and data platform supported in [dbt Cloud](/docs/cloud/connect-data-platform/about-connections).
-- You are using dbt version 1.0 or later.
+- A supported version of dbt or select [Keep on latest version](/docs/dbt-versions/upgrade-dbt-version-in-cloud#keep-on-latest-version) of dbt. <Lifecycle status="Preview"/>
```

Comment:
Maybe this is a decision @jtcohen6, but this wording might be a little misleading. Our dbt Cloud environments still allow people to select 1.0. Nothing in their environment tells them that 1.0-1.4 aren't supported. I think we should not only link to our supported versions doc, but make it explicitly clear that dbt Cloud allows the customer to select versions that are no longer supported.

Reply:
@matthewshaver I think I agree with the thrust of what you're saying. Starting with the GA in early May, we are going to make "Keep on latest version" the default for every new customer, and strongly encourage every existing customer to upgrade their environments.

A few months from now, my inclination would be to cut mention of dbt versions section entirely. As long as you're not using a _very old_ version (pre-1.0), your project code should run successfully in dbt Cloud on "latest."